core/iwasm/libraries/debug-engine/gdbserver.c

```diff
@@ -180,13 +134,13 @@ handler_packet(WASMGDBServer *server, char request, char *payload)
 static void
 process_packet(WASMGDBServer *server)
 {
-    uint8_t *inbuf = server->pkt.buf;
-    int inbuf_size = server->pkt.size;
-    uint8_t *packetend_ptr = (uint8_t *)memchr(inbuf, '#', inbuf_size);
-    int packetend = packetend_ptr - inbuf;
+    uint8 *inbuf = server->pkt.buf;
+    int32 inbuf_size = server->pkt.size;
+    uint8 *packetend_ptr = (uint8 *)memchr(inbuf, '#', inbuf_size);
+    int32 packetend = (int32)(uintptr_t)(packetend_ptr - inbuf);
```

Comment:
How about rename packetend to packet_size?

Reply:
Done